lib/bgsite_official_web/templates/feedback/new.html.eex

```diff
@@ -0,0 +1,8 @@
+<div>
+<h1>Обратна връзка</h1>
+
+
+<%= render "form.html", Map.put(assigns, :action, Routes.feedback_path(@conn, :create)) %>
+
+<!-- <span><%= link "Back", to: Routes.feedback_path(@conn, :index) %></span> -->
```

Comment:
This row would cause exceptions no matter it is commented, we have to be careful.